health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/FacilityPopup.js

```
@@ -325,6 +326,93 @@ const FacilityPopUp = ({ details, onClose, updateDetails }) => {
     setLoader(false);
   };
 
+  const handleAssignment = async () => {
+    // Fetching the full data of selected rows
+    setLoader(true);
+    const selectedRowData = censusData.filter(row => selectedRows.includes(row.id));
+    var newDetails = JSON.parse(JSON.stringify(details));
```

Biome

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole body of the function. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'const' instead.


(lint/style/noVar)